pkg/controllers/workapplier/status.go

```diff
@@ -151,24 +148,27 @@ func (r *Reconciler) refreshWorkStatus(
 		if bundle.availabilityResTyp == ManifestProcessingAvailabilityResultTypeNotTrackable {
 			untrackableAppliedObjectsCount++
 		}
+		if isManifestObjectDiffReported(bundle.reportDiffResTyp) {
+			diffReportedObjectsCount++
+		}
```

Comment:
Why Is there no UT test change related to this change that count "both diff and noDiff as reported"

Reply:
Additional UTs added.